src/applications/personalization/profile/tests/e2e/contact-information/modals.cypress.spec.js

```diff
@@ -90,18 +92,26 @@ const checkRemovalWhileEditingModal = options => {
   });
 
   // Modal appears
-  cy.get('.va-modal').within(() => {
-    cy.contains(`You’re currently editing your ${editSectionName}`).should(
-      'exist',
-    );
-    cy.findByRole('button', { name: /OK/i }).click({
-      force: true,
-    });
-  });
+  // cy.get('.va-modal').within(() => {
+  //   cy.contains(`You’re currently editing your ${editSectionName}`).should(
+  //     'exist',
+  //   );
+  //   cy.findByRole('button', { name: /OK/i }).click({
+  //     force: true,
+  //   });
+  // });
```

Comment:
Now that we have fixed the tests, we should remove these commented out sections of code

Reply:
Of course, sorry I missed it.